content/attacks/2023-11-01- KANDYKORN

```diff
@@ -0,0 +1,59 @@
+---
+date: 2023-11-01
+target-entities: Blockchain engineers of an unnamed crypto exchange platform.
+entity-types:  
+- Blockchain
+- Engineers
+- Chat Channel
+- Exchange
+attack-types:
+- Social engineering   
+- Execution flow hijacking
+- defensive evasion techniques
+- KANDYKORN
+- Swift-based self-signed binary
+- APT-43
+title: "North Korean Hackers Targeting Crypto Experts with KANDYKORN macOS Malware."
+loss: No monetary loss
+---
+
+## Summary
```

Comment:
a little bit too much discourse/discussion. focus the summary on the hack/exploit, losses, entities involved.  No need to include extended discussion on peripherals.